service/dataset.py

```diff
@@ -23,17 +23,11 @@
                         },
                         "budget":3.0}}
 
+RELEASED_DATASETS = []
```

Comment:
We should also add one to start as an example

Reply:
example added everywhere